ztp/siteconfig-generator-kustomize-plugin/testSiteConfig/site2-sno-du.yaml

```diff
@@ -31,6 +31,10 @@ spec:
       - NTP.server1
       - 10.16.231.22
     ignitionConfigOverride: "igen"
+    diskEncryption:
+      type: "tpm2"
+      tpm2:
+        pcrList: "1,8"
```

Comment:
I think we need to add more sites
so we can test all 4 options (none, tpm2, nbde, tpm2+nbde)

Reply:
fixed